src/Resources/contao/library/Contao/Database.php

```diff
@@ -707,6 +707,36 @@ public function getUuid()
 
 
 	/**
+	 * Escape the column name if it a reserved word
+	 *
+	 * @param string $name
+	 *
+	 * @return string
+	 */
+	public static function escapeColumnName($name)
+	{
+		if (in_array($name, ['rows'], true)) {
```

Comment:
Can't we use the doctrine platform method here to quote everything we need (plus force `rows`)?

Reply:
Isn't `in_array($name, ['rows'], true)` the same as `'rows' === $name`? Or did you just want to avoid the Yoda condition. 😄

Reply:
He just prepared for more in the future.

Reply:
@ausi, what about this? (should work in 3.5 too):



```php
// I'd name it "quote", not "escape"...
public static function quoteColumnName($name)
{
    if (method_exists('Contao\System', 'getContainer')) {
        $connection = Contao\System::getContainer()->get('connection');
        if ($connection instanceof Doctrine\DBAL\Connection) {
            $name = $connection->getDatabasePlatform()->quoteSingleIdentifier($name);
        }
    }

    // Force quoting "rows" for older Contao versions
    // (which will be ignored if already done by the platform in newer Contao versions)
    if (in_array($name, ['rows'], true)) {
        $name = '`'.$name.'`';
    }

    return $name;
}
```